### DISCUSSION TITLE
Welcome to awesome-resources Discussions!

### DISCUSSION THREAD

**Discussion by shahednasser** [shahednasser/awesome-resources#150] [Announcements]
## 👋 Welcome!
  
Thank you for your interest in contributing to awesome-resources! If you have questions about what kind of resources can be added, what kind of topics, how to contribute, or any other kind of questions, please add them here.

Thank you for your time!

**Comment by richernote** [comment]
I would like to go through some of the resources and split them up. An example is under "Laravel" there's the whole Laracast.com site but it has a whole bunch of courses on laravel, php, vue etc. I would like to separate that resource into its individual parts.

**Reply by shahednasser** [reply]
That's interesting. Would you mind either sending a PR with it or clarify your idea more? I think it makes sense but we also have to take into account that there are different categories and we should choose a way that would be uniform to all categories.
